pkg/internal/reporter/reporter_test.go

```diff
@@ -51,23 +51,22 @@ func TestReportShouldSucceed(t *testing.T) {
 }
 
 func TestReportShouldSuccessfullyReportResults(t *testing.T) {
-	const (
-		expectedTrafficGenSentPackets        = 0
-		expectedTrafficGenOutputErrorPackets = 0
-		expectedTrafficGenInputErrorPackets  = 0
-		expectedVMUnderTestReceivedPackets   = 0
-		expectedVMUnderTestRxDroppedPackets  = 0
-		expectedVMUnderTestTxDroppedPackets  = 0
-		expectedVMUnderTestActualNodeName    = "dpdk-node01"
-		expectedTrafficGenActualNodeName     = "dpdk-node02"
-	)
-
 	const (
 		failureReason1 = "some reason"
 		failureReason2 = "some other reason"
 	)
 
 	t.Run("on checkup success", func(t *testing.T) {
+		const (
+			expectedTrafficGenSentPackets        = 0
+			expectedTrafficGenOutputErrorPackets = 0
+			expectedTrafficGenInputErrorPackets  = 0
+			expectedVMUnderTestReceivedPackets   = 0
+			expectedVMUnderTestRxDroppedPackets  = 0
+			expectedVMUnderTestTxDroppedPackets  = 0
```

Comment:
WDYT about removing these and rely on the fact that this is the default zero value?

Reply:
I want to be explicit.